stan/math/prim/mat/fun/dot_product.hpp

```diff
@@ -19,8 +19,8 @@ namespace math {
  * size or if they are both not vector dimensioned.
  */
 template <int R1, int C1, int R2, int C2>
-inline double dot_product(const Eigen::Matrix<double, R1, C1>& v1,
-                          const Eigen::Matrix<double, R2, C2>& v2) {
+inline double dot_product(const Eigen::Matrix<double, R1, C1> &v1,
```

Comment:
Only a stylistic point, but the previous formatting is consistent with the rest of the codebase

Reply:
The formatting's all handled automatically now, I believe.